Xamarin.Forms.Platform.UAP/FormsTextBox.cs

```diff
@@ -176,12 +178,16 @@ protected override void OnApplyTemplate()
 			_scrollViewer= (Windows.UI.Xaml.Controls.ScrollViewer)GetTemplateChild("ContentElement");
 		}
 
+		void OnSizeChanged(object sender, SizeChangedEventArgs e)
+		{
+			UpdateTemplateScrollViewerVerticalAlignment();
```

Comment:
@bmacombe Does moving the method here resolve this issue and still keep this one fixed https://github.com/xamarin/Xamarin.Forms/pull/11140. as far as you can tell? 

I think the problem is happening because for whatever reason if you set the _scrollViewer VerticalAlignment before the TextBox has a size then it doesn't render correctly for whatever reason 

Also, I don't think we want to call `Packager.Load();` from the Loaded event. That just feels really late to me in the life cycle to add/create all the child renderers. Loading? perhaps but Loaded feels too late

Reply:
@PureWeen  I just manually looked at all the issue pages (2172,8787,8503,8222) that I was using to verify all these related fixes and it all looks good!

You might be right about using Loading vs Loaded, but I'm actually very relieved that the call to loaded can be removed.  I've been trying to avoid adding that back since working on 2172 because it appears to alter how the UWP layout has been working for a while now and I really don't want to break anybody...to much stress, don't know how you full-time platform maintainers do it!  I vote to just leave it alone until there is a need to move Packager.Load to later in the layout cycle since it doesn't appear to be needed for this issue.

Thanks for the assist!  These entry size and text issues have been hard to track down and it's great we might finally have them all taken care of!

I'll do a little clean up of comments, etc, test it all again and then push up a commit and fingers crossed this should be good to merge!

Reply:
I'll rebase and clean up #11285 to be merged after this PR also, there probably will be some conflicts in the comments.  Thought about just putting those changes in this PR also, but maybe it's good to keep the issues separate?  I'll go that route unless I hear different from the team

Reply:
> These entry size and text issues have been hard to track down and it's great we might finally have them all taken care of!

Yea it's such an annoying balance of timings on all of this :-/

> I'll do a little clean up of comments, etc, test it all again and then push up a commit and fingers crossed this should be good to merge!

Thank you!! It looks like the tests have all passed. There's a ScrollView one that's also failing occasionally on the main branch. 👎 but otherwise looks good